ember-aria-utilities/src/test-support/-private/aria-grid.ts

```diff
@@ -150,8 +151,23 @@ const keys = {
   f2: triggerable(F2),
 } as const;
 
+let notified = false;
+
 export const ariaGrid = {
   keys,
-  gridSelectors,
+  get selectors() {
+    if (!notified) {
+      deprecate('selectors are being deprecated, use gridSelectors instead', false, {
+        id: '@crowdstrike/ember-aria-utilities/test-support',
+        until: 'eventually',
```

Comment:
lol, eventually 🙃

Reply:
yah I should update that to something more specific you are right...